src/main/java/xyz/nucleoid/creator_tools/command/MapMetadataCommand.java

```diff
@@ -79,25 +166,25 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                     )))))
                     .then(literal("rename")
                         .then(literal("all")
-                            .then(argument("old", StringArgumentType.word()).suggests(regionSuggestions())
-                            .then(argument("new", StringArgumentType.word())
+                            .then(regionMarkerArg("old")
```

Comment:
If the 'exact' marker is extracted, it might help to rename this to 'region selector'/'region predicate' kind of thing

Reply:
I don't quite follow unfortunately - perhaps the attached diff is wrong though